providers/edge3/src/airflow/providers/edge3/worker_api/routes/ui.py

```diff
@@ -17,18 +17,21 @@
 
 from __future__ import annotations
 
-from fastapi import Depends
+from datetime import datetime
+
+from fastapi import Depends, HTTPException
 from sqlalchemy import select
 
 from airflow.api_fastapi.auth.managers.models.resource_details import AccessView
 from airflow.api_fastapi.common.db.common import SessionDep  # noqa: TC001
 from airflow.api_fastapi.common.router import AirflowRouter
 from airflow.api_fastapi.core_api.security import requires_access_view
```

Comment:
Replication of commit https://github.com/apache/airflow/pull/55348/commits/ec77bd65b96fd27eb601aeda51d4fb92354bc301 (1)
```suggestion
from airflow.api_fastapi.core_api.security import GetUserDep, requires_access_view
```